### DISCUSSION TITLE
Can vanna-ai LLM train for Trino funcutions

### DISCUSSION THREAD

**Discussion by cw1427** [vanna-ai/vanna#290] [Q&A]
It seems vanna-ai LLM model can generate original SQL DDL query, can it be possible to train with Trino functions? That has many useful aggregation funcs for DBs   https://trino.io/

**Comment by zainhoda** [comment]
Could you provide an example of something that you're currently unable to do?

**Comment by njalan** [comment]
@zainhoda I faced one issue is that it date function is not used properly in vanna(I am using trino).  For example: I am asking data for one specific day but in sql it use one date ranger. Sometimes I asked for one month but in sql it use date function for one day or many  months. Is there any good practice to train for date related questions?
